api_tests/lib_sdk/lnd.ts

```diff
@@ -131,8 +133,26 @@ export class Lnd {
         await sleep(1);
     }
 
-    public async fund() {
-        await sleep(1);
+    public async createChainAddress(): Promise<string> {
+        const response = await lnService.createChainAddress({
+            format: "np2wpkh",
+            lnd: this.authenticatedLndGrpc,
+        });
+        return response.address;
+    }
+
+    public async getChainBalance(): Promise<number> {
+        return (
+            await lnService.getChainBalance({ lnd: this.authenticatedLndGrpc })
+        ).chain_balance;
+    }
+
+    public async getChannelBalance(): Promise<number> {
```

Comment:
Shouldn't this take an identifier of the channel or something?

Reply:
Nope.